gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/test/java/io/gravitee/gateway/jupiter/reactor/processor/DefaultPlatformProcessorChainFactoryTest.java

```diff
@@ -69,91 +72,95 @@ public void setup() {
         lenient().when(transactionHandlerFactory.create()).thenReturn(mock(TransactionProcessor.class));
     }
 
-    @Test
-    @DisplayName("Should have 4 pre processors when trace context is enabled")
-    public void shouldHave3PreProcessorsWhenTraceContextEnabled() {
-        DefaultPlatformProcessorChainFactory platformProcessorChainFactory = new DefaultPlatformProcessorChainFactory(
-            transactionHandlerFactory,
-            true,
-            reporterService,
-            eventProducer,
-            node,
-            "8080",
-            false,
-            gatewayConfiguration
-        );
-        List<Processor> processors = platformProcessorChainFactory.buildPreProcessorList();
-
-        assertEquals(4, processors.size());
-        assertTrue(processors.get(0) instanceof MetricsProcessor);
-        assertTrue(processors.get(1) instanceof XForwardForProcessor);
-        assertTrue(processors.get(2) instanceof TraceContextProcessor);
-        assertTrue(processors.get(3) instanceof TransactionProcessor);
+    @Nested
+    class PreProcessors {
+
+        @Test
+        void should_have_preProcessors_with_TraceContextProcessor_when_traceContext_is_not_enabled() {
```

Comment:
I think the label of the test is incorrect.

```suggestion
        void should_have_preProcessors_with_TraceContextProcessor_when_traceContext_is_enabled() {
```